src/content/en/fundamentals/getting-started/primers/media-source-extensions.md

```diff
@@ -58,11 +58,12 @@ In practice, the chain looks like this:
           return response.arrayBuffer();
         })
         .then(function(arrayBuffer) {
-          sourceBuffer.onupdateend = function(e) {
-            if (sourceBuffer.updating && mediaSource.readyState === 'open') {
+          sourceBuffer.addEventListener('updateend', function(e) {
+            if (!sourceBuffer.updating && mediaSource.readyState === 'open') {
               mediaSource.endOfStream();
+              // window.URL.revokeObjectURL(vidElement.src);
```

Comment:
1. Please remove `//` comment.
2. For consistency, let's use `URL.revokeObjectURL` instead of `window.URL.revokeObjectURL`